integration/vm_factory/vm_cache_test.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 ARM Limited
```

Comment:
2020 ?

Reply:
haha,,, I've fired this PR in 2019. I'll update asap~~~~